app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts
+    @posts = @user.posts.includes(:comments)
     @current_user = current_user
```

Comment:
- [ ] Please add screenshots of the logs before and after the fix to the PR description